polaris-react/playground/KitchenSink.tsx

```diff
@@ -7,8 +7,9 @@ interface Stories {
 }
 
 const stories: Stories = {};
+// @ts-expect-error require.context is a Webpack-specific thing
 const req = require.context('../src/components', true, /.stories.tsx$/);
-req.keys().forEach((filePath) =>
+req.keys().forEach((filePath: string) =>
```

Comment:
Don't know how these TS failures made it to `main`, but I wasn't able to build the site locally without fixing them 🤷